pkg/kubelet/dockershim/helpers_linux_test.go

```diff
@@ -55,26 +58,26 @@ func TestGetSeccompSecurityOpts(t *testing.T) {
 }
 
 func TestLoadSeccompLocalhostProfiles(t *testing.T) {
+	tmpdir, err := ioutil.TempDir("", "seccomp-local-profile-test")
+	assert.NoError(t, err)
+	defer os.RemoveAll(tmpdir)
+	testProfile := `{"foo": "bar"}`
+	err = ioutil.WriteFile(filepath.Join(tmpdir, "test"), []byte(testProfile), 0644)
+	assert.NoError(t, err)
+
 	tests := []struct {
 		msg            string
 		seccompProfile string
 		expectedOpts   []string
 		expectErr      bool
 	}{{
-		msg: "Seccomp localhost/test profile",
-		// We are abusing localhost for loading test seccomp profiles.
-		// The profile should be an absolute path while we are using a relative one.
-		seccompProfile: "localhost/fixtures/seccomp/test",
+		msg:            "Seccomp localhost/test profile",
+		seccompProfile: "localhost/" + filepath.Join(tmpdir, "test"),
 		expectedOpts:   []string{`seccomp={"foo":"bar"}`},
 		expectErr:      false,
-	}, {
-		msg:            "Seccomp localhost/sub/subtest profile",
-		seccompProfile: "localhost/fixtures/seccomp/sub/subtest",
-		expectedOpts:   []string{`seccomp={"abc":"def"}`},
-		expectErr:      false,
 	}, {
 		msg:            "Seccomp non-existent",
-		seccompProfile: "localhost/fixtures/seccomp/non-existent",
+		seccompProfile: "localhost/" + filepath.Join(tmpdir, "fixtures/non-existent"),
```

Comment:
should have separate tests for does-not-exist and relative paths

Reply:
ack